app/main/views/organisations.py

```diff
@@ -156,10 +158,50 @@ def organisation_dashboard(org_id):
         **{
             f'total_{key}': sum(service[key] for service in services)
             for key in ('emails_sent', 'sms_cost', 'letter_cost')
-        }
+        },
+        download_link=url_for(
+            '.download_organisation_usage_report',
+            org_id=org_id,
+            selected_year=year
+        )
     )
 
 
+@main.route("/organisations/<uuid:org_id>/download-usage-report.csv", methods=['GET'])
+@user_has_permissions()
+def download_organisation_usage_report(org_id):
+    selected_year = request.args.get('selected_year')
+    services_usage = current_organisation.services_and_usage(
+        financial_year=selected_year
+    )['services']
+
+    column_names = OrderedDict([
+        ('service_id', 'Service ID'),
+        ('service_name', 'Service Name'),
+        ('emails_sent', 'Emails sent'),
+        ('sms_remainder', 'Free text message allowance remaining'),
+        ('sms_cost', 'Spent on text messages'),
+        ('letter_cost', 'Spent on letters')
```

Comment:
Optional suggestion to make it clear what units we‘re using:
```suggestion
        ('letter_cost', 'Spent on letters (£)')
```